packages/itwinui-css/src/surface/surface.scss

```diff
@@ -22,5 +22,43 @@
 
 // Public facing surface component.
 @mixin iui-surface-component {
+  --iui-surface-padding-inline: var(--iui-size-s);
+  --iui-surface-body-padding: var(--iui-surface-padding-inline);
```

Comment:
Do these need to be configurable? I don't expect people changing this except to remove the body padding entirely.
```jsx
<Surface>
  <Surface.Body padded>...</Surface>
</Surface>
```
```jsx
<Surface>
  <Surface.Body>...</Surface>
</Surface>
```

Reply:
I've changed `--iui-surface-padding-inline` to `--_iui-surface-padding-inline` & body padding is now set via `data-iui-padded="true"` on `.iui-surface`.

Reply:
It doesn't need to be that complicated. Unless there is a strong reason for it, I think we can get rid of both of these CSS variables entirely. If you like the readability benefit, then using Sass variables is perfectly fine (it will get compiled out).

Reply:
Now using a Sass variable for padding.